mapper.html

```diff
@@ -0,0 +1,204 @@
+        </div>
+        <div class="pre">
+          received <
+          {"method":"Target.targetCreated","params":{"targetInfo":{"targetId":"C51F49DEFB9019E642F4E073EBD4F777","type":"page","title":"about:blank","url":"about:blank","attached":false,"canAccessOpener":false,"browserContextId":"46F06D6AE4D902E6D85F6350C1CB607D"}}}
+        </div>
+        <div class="pre">received < {"id":0,"result":{}}</div>
+        <div class="pre">
+          sent >
+          {"id":1,"method":"Target.setAutoAttach","params":{"autoAttach":true,"waitForDebuggerOnStart":true,"flatten":true}}
+        </div>
+        <div class="pre">
+          received <
+          {"method":"Target.attachedToTarget","params":{"sessionId":"8309F5AB91D1C806AF4F11DC221C341B","targetInfo":{"targetId":"18A8DDD677D77AC724D8F4A1977D89AE","type":"page","title":"BiDi
+          Mapper","url":"about:blank","attached":true,"canAccessOpener":false,"browserContextId":"46F06D6AE4D902E6D85F6350C1CB607D"},"waitingForDebugger":false}}
+        </div>
+        <div class="pre">
+          sent >
+          {"id":2,"method":"Runtime.runIfWaitingForDebugger","sessionId":"8309F5AB91D1C806AF4F11DC221C341B"}
+        </div>
+        <div class="pre">
+          received <
+          {"method":"Target.targetInfoChanged","params":{"targetInfo":{"targetId":"C51F49DEFB9019E642F4E073EBD4F777","type":"page","title":"about:blank","url":"about:blank","attached":true,"canAccessOpener":false,"browserContextId":"46F06D6AE4D902E6D85F6350C1CB607D"}}}
+        </div>
+        <div class="pre">
+          received <
+          {"method":"Target.attachedToTarget","params":{"sessionId":"FE8CDA12CEA66EB00241D6F8B87D9743","targetInfo":{"targetId":"C51F49DEFB9019E642F4E073EBD4F777","type":"page","title":"about:blank","url":"about:blank","attached":true,"canAccessOpener":false,"browserContextId":"46F06D6AE4D902E6D85F6350C1CB607D"},"waitingForDebugger":false}}
+        </div>
+        <div class="pre">
+          sent >
+          {"id":3,"method":"Runtime.enable","sessionId":"FE8CDA12CEA66EB00241D6F8B87D9743"}
+          received < {"id":1,"result":{}}
+        </div>
+        <div class="pre">
+          received <
+          {"method":"Runtime.executionContextCreated","params":{"context":{"id":1,"origin":"://","name":"","uniqueId":"-7179749106418105585.-1327732253193402396","auxData":{"isDefault":true,"type":"default","frameId":"C51F49DEFB9019E642F4E073EBD4F777"}}},"sessionId":"FE8CDA12CEA66EB00241D6F8B87D9743"}
+        </div>
+        <div class="pre">
+          received <
+          {"id":3,"result":{},"sessionId":"FE8CDA12CEA66EB00241D6F8B87D9743"}
+        </div>
+        <div class="pre">
+          sent >
+          {"id":4,"method":"Page.enable","sessionId":"FE8CDA12CEA66EB00241D6F8B87D9743"}
+        </div>
+        <div class="pre">
+          received <
+          {"id":2,"result":{},"sessionId":"8309F5AB91D1C806AF4F11DC221C341B"}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<style>
```

Comment:
Let’s move the `<style>` to the top of the doc (in the `<head>`)

Reply:
done